src/org-pending/components/summary/summary.component.html

```diff
@@ -0,0 +1,78 @@
+<app-identity-bar-component><span>{{data.name}} </span><span class="hmcts-badge">{{data.status}}</span></app-identity-bar-component>
+<div class="govuk-width-container">
+<div class="govuk-main-wrapper">
+<div class="govuk-heading-xl">
```

Comment:
<div class="govuk-width-container">
<div class="govuk-main-wrapper"> 
should live in the compoents that wrapps compoent with ng-content

Reply:
done